web-admin/src/features/alerts/metadata/AlertMetadata.svelte

```diff
@@ -41,9 +41,7 @@
   $: dashboardName = useAlertDashboardName($runtime.instanceId, alert);
   $: dashboard = useDashboard($runtime.instanceId, $dashboardName.data);
   $: dashboardTitle =
-    $dashboard.data?.metricsView.spec.title ||
-    $dashboardName.data ||
-    $alertQuery.data?.resource?.meta?.name?.name;
+    $dashboard.data?.metricsView.spec.title || $dashboardName.data;
```

Comment:
A general note – given [this convo](https://rilldata.slack.com/archives/CTCJ58H3M/p1725629184591609?thread_ts=1725577591.865249&cid=CTCJ58H3M), ultimately we should migrate from `metricsView.spec.[attribute]` to `metricsView.state.validSpec.[attribute]`